azure/notebooks/kubernetes/MNMG_XGBoost.ipynb

```diff
@@ -0,0 +1,912 @@
+{
```

Comment:
<p>And link AKS</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cloud-ml-examples/pull/118/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>